front/SettingsForm.tsx

```diff
@@ -72,6 +72,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="text"
               placeholder="Username"
+              data-cy="settings-username"
```

Comment:
The data-cy attribute value for the username input field should be 'settings-username-input' for better clarity.